yarn.lock

```diff
@@ -3176,23 +3192,34 @@ vite@^4.0.0, vite@^4.4.6:
   optionalDependencies:
     fsevents "~2.3.2"
 
-vitepress@^1.0.0-beta.2:
-  version "1.0.0-beta.6"
-  resolved "https://registry.yarnpkg.com/vitepress/-/vitepress-1.0.0-beta.6.tgz#f5439bcbb53ab9a51f6f2b2a57d1bfad19cff476"
-  integrity sha512-xK/ulKgQpKZVbvlL4+/vW49VG7ySi5nmSoKUNH1G4kM+Cj9JwYM+PDJO7jSJROv8zW99G0ise+maDYnaLlbGBQ==
+vite@^4.4.9:
```

Comment:
Looks like we’ve picked up multiple copies of rollup and vite in this upgrade. This is why I generally prefer to upgrade everything at once via `yarn upgrade` rather than doing it piecemeal. I’ll do that now and push to this branch.